tripleblindmarket/covid-safe-paths
# Put auto sub behind dev flags

Signed-off-by: Patrick Erichsen <patrick.a.erichsen@gmail.com>

## Description

Puts the Auto Subscribe feature for HAs behind a dev feature flag.

#### Screenshots:

<img width="399" alt="Screen Shot 2020-04-28 at 6 28 16 PM" src="https://user-images.githubusercontent.com/20157849/80547529-13d78f00-897e-11ea-99d8-bc874f91135a.png">
<img width="413" alt="Screen Shot 2020-04-28 at 6 25 53 PM" src="https://user-images.githubusercontent.com/20157849/80547531-1508bc00-897e-11ea-815a-e1068d81d8e6.png">
<img width="415" alt="Screen Shot 2020-04-28 at 8 22 36 PM" src="https://user-images.githubusercontent.com/20157849/80553028-0c1fe680-898e-11ea-85ff-73a6fdb7f6a6.png">

#### How to test

- Run through onboarding in both iOS and Android and confirm that the HA auto sub features are not visible.
- Visit the `ChooseProvider` screen and confirm the auto sub checkbox is not visible
